### PR TITLE
Small DocBlock update

### PR DESCRIPTION
**Pull request by sreichel**
... detected with strict types on.

### CODE REVIEW

File: app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php [hunk 1]
```diff
@@ -231,7 +231,7 @@ protected function findInMapping($frontModule, $controllerName, $actionName)
     /**
      * Determine which version of docs should target onto
      *
-     * @return string
+     * @return false|Varien_Simplexml_Element
```
Comment (kiatng) [inline app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php:234]:
```suggestion
     * @return Mage_Core_Model_Config_Element|Varien_Simplexml_Element|false
```

File: app/code/core/Mage/Core/Model/Session/Abstract.php [hunk 2]
```diff
@@ -524,7 +524,7 @@ public function getSessionHosts()
     /**
      * Retrieve session save method
      *
-     * @return string
+     * @return string|Varien_Simplexml_Element
```
Comment (kiatng) [inline app/code/core/Mage/Core/Model/Session/Abstract.php:527]:
```suggestion
     * @return Mage_Core_Model_Config_Element|Varien_Simplexml_Element|false|string
```

File: app/code/core/Mage/Core/Model/Session/Abstract.php [hunk 3]
```diff
@@ -537,7 +537,7 @@ public function getSessionSaveMethod()
     /**
      * Get session save path
      *
-     * @return string
+     * @return string|Varien_Simplexml_Element
```
Comment (kiatng) [inline app/code/core/Mage/Core/Model/Session/Abstract.php:540]:
```suggestion
     * @return Mage_Core_Model_Config_Element|Varien_Simplexml_Element|false|string
```

File: app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php:234]: Thought about it ....

I thinks with strict types on phpstan needs a class that implements `__toString()`.

`Mage_Core_Model_Config_Element` extends `Varien_Simplexml_Element` so i used this.

To narrow it down, how about `Mage_Core_Model_Config_Element|false`?

(but yes, add all types from parent method is also okay)

**Review by kiatng — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Pull request merged**